region/maridia/inner-pink/East Sand Hole.json

```diff
@@ -155,9 +146,29 @@
               "canSunkenTileWideWallClimb"
             ]}
           ]},
+          "canSpringBallJumpMidAir",
+          {"and": [
+            "h_canCrouchJumpDownGrab",
+            "canBombJumpWaterEscape",
+            "h_canIBJ"
```

Comment:
```suggestion
            "h_canJumpIntoIBJ"
```

Reply:
Is this possible to do with a single jump? I can't get it

Reply:
This is a funny way to look at it, but I agree with your reasoning.

With HiJump it looks like your jumping out of the water, and without HJB you are bomb jumping at the water line so they look like they should be switched.